src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -801,5 +851,26 @@ public void Dispose()
         {
             Dispose(disposing: true);
         }
+
+        /// <inheritdoc />
+        public async ValueTask DisposeAsync()
+        {
+            if (_disposeTask != null)
```

Comment:
Can we use `_disposed` here to determine if this was previously invoked and no-op?

Reply:
This was done, not sure why it doesn't show